integration_tests/suite/base/test_switchboards.py

```diff
@@ -214,11 +214,17 @@ def test_edit_minimal_parameters(switchboard):
     assert_that(response.item, has_entries(name='after_edit'))
 
 
-@fixtures.moh(name='foo')
-@fixtures.switchboard(
-    queue_music_on_hold='foo', waiting_room_music_on_hold='foo', timeout=42
-)
-def test_update_fields_with_null_value(_, switchboard):
+@fixtures.moh(label='foo')
+def test_update_fields_with_null_value(moh):
+    response = confd.switchboards.post(
+        name='TheSwitchboard',
+        queue_music_on_hold=moh['name'],
+        waiting_room_music_on_hold=moh['name'],
+        timeout=42,
+    )
```

Comment:
You should remove switchboard at the end of the test to avoid any collision

Reply:
Good catch